gulpfile.js

```diff
@@ -41,7 +41,7 @@ gulp.task('clean', function () {
 
 gulp.task('devpack', function () {
   webpackConfig.devtool = 'source-map';
-  return gulp.src('src/*.js')
+  return gulp.src(['src/prebid.js', 'src/*.js'])
```

Comment:
Is the 2nd argument needed, or can webpack find all the required source through `require` in `prebid.js` ?